src/test/java/io/logz/LogzioSenderTest.java

```diff
@@ -378,4 +380,40 @@ public void testMDC() throws Exception {
 
         assertThat(logRequest.getStringFieldOrNull(mdcKey)).isEqualTo(mdcValue);
     }
+
+    @Test
+    public void testContextReset() throws Exception {
```

Comment:
👍 
Maybe comment above: context.reset() is called when logback loads a new logback.xml in-flight